content/en/docs/distributions/aws/troubleshooting-aws.md

```diff
@@ -1,64 +1,62 @@
 ```
 
-If you see this error, you probably didn't use `cluster_name` as folder name during setup. Please go to check your cluster setting by `kubectl get configmaps aws-alb-ingress-controller-config -n kubeflow -o yaml` and make the change. Another reason could be that you did not tag your subnets  so that Kubernetes knows to use only those subnets for external load balancers. To fix this ensure the public subnets are tagged with the **Key**: ```kubernetes.io/role/elb``` and **Value**: ```1```. See docs [here](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html) for further details.
-
-### Kubeflow Uninstallation Failure
+If you see this error, you likely didn't install from a directory named `cluster_name` during setup. Please check `kubectl get configmaps aws-alb-ingress-controller-config -n kubeflow -o yaml` and make any needed change.
 
+Another reason could be that your subnets are not tagged so that Kubernetes knows which subnets to use for external load balancers. To fix this, ensure your cluster's public subnets are tagged with the **Key**: ```kubernetes.io/role/elb``` and **Value**: ```1```. See the [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html) for further details.
 
-```
-Error: couldn't delete KfApp:  (kubeflow.error): Code 500 with message: kfApp Delete failed for kustomize:  (kubeflow.error): Code 500 with message: error deleting kustomize manifests: [error evaluating kustomization manifest for knative: Timed out waiting for resource /knative-serving to be deleted. Error deleted resource is not cleaned up yet, error evaluating kustomization manifest for cert-manager: Timed out waiting for resource /cert-manager to be deleted. Error deleted resource is not cleaned up yet]
-Usage:
-  kfctl delete [flags]
-
-Flags:
-      --delete_storage   Set if you want to delete app's storage cluster used for mlpipeline.
-  -f, --file string      The local config file of KfDef.
-      --force-deletion   force-deletion output default is false
-  -h, --help             help for delete
-  -V, --verbose          verbose output default is false
-
-kfctl exited with error: couldn't delete KfApp:  (kubeflow.error): Code 500 with message: kfApp Delete failed for kustomize:  (kubeflow.error): Code 500 with message: error deleting kustomize manifests: [error evaluating kustomization manifest for knative: Timed out waiting for resource /knative-serving to be deleted. Error deleted resource is not cleaned up yet, error evaluating kustomization manifest for cert-manager: Timed out waiting for resource /cert-manager to be delete
+### Resource Not Found in `delete all`
```

Comment:
I feel like you can simply delete this section since it's basically K8s type of failure.

Reply:
I was assuming it was in there to let users know they can ignore. Happy to remove if that's ok.